modules/linux_packages/0.0.5/MODULE.bazel

```diff
@@ -0,0 +1,50 @@
+"Bazel dependencies"
+
+module(
+    name = "linux_packages",
```

Comment:
nit: should it be `bazel_linux_packages` to match the repo name?

Reply:
I find the `bazel` prefix inside the bazel ecosystem superfluous. But`linux_packages` as a repo name a little to generic :)

Reply:
Indeed that's my concern, so I still slightly prefer adding the prefix. There are many modules with "bazel" in their name.

Reply:
Done, @meteorcloudy  please close this PR for https://github.com/bazelbuild/bazel-central-registry/pull/3980